packages/monaco-editor-wrapper/src/editor.ts

```diff
@@ -106,31 +106,33 @@ export class MonacoEditorBase {
             return Promise.reject(new Error('You cannot update the editor model, because the regular editor is not configured.'));
         }
 
-        if (modelUpdate.code != undefined) {
+        if (modelUpdate.code !== undefined) {
```

Comment:
One last thing: Please extract these checks into a separate function, then it can be used by `updateModel` and `updateDiffModel`.